vyxal/LazyList.py

```diff
@@ -165,9 +165,7 @@ def __next__(self):
         return item
 
     def __setitem__(self, position, value):
```

Comment:
We can just get rid of this method tbh, no need to implement it and raise a `NotImplementedError`

Reply:
To be removed later when we're sure we got every instance of mutability.

For now, it's a failsafe.